bindings/wasm/package.json

```diff
@@ -15,16 +15,18 @@
   "exports": {
     ".": {
       "node": "./node/dist/index.cjs",
-      "browser": "./web/dist/index.js",
+      "browser": "./web/dist/interface.js",
```

Comment:
Should `default` also be `interface.js`? Further, perhaps this should be sth like

```
      "node": {
        "import": "./node/dist/index.mjs", // if we have ES module support in node, i dont remember
        "require": "./node/dist/index.cjs",
      },
      "browser": {
        "import": "./web/dist/interface.js",
        "require": "???"
      }
```